packages/salesforcedx-utils-vscode/test/unit/commands/commandletExecutor.test.ts

```diff
@@ -0,0 +1,239 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { expect } from 'chai';
+import { EOL } from 'os';
+import * as proxyquire from 'proxyquire';
+import { assert, createSandbox, SinonSandbox, SinonStub, stub } from 'sinon';
+import { nls } from '../../../src/messages';
+import { ContinueResponse } from '../../../src/types';
+
+class MockChannel {
+  public readonly name = 'MockChannel';
+  public value = '';
+
+  public append(value: string): void {
+    this.value += value;
+  }
+
+  public appendLine(value: string): void {
+    this.value += value;
+    this.value += EOL;
+  }
+
+  public clear(): void {}
+  public show(preserveFocus?: boolean | undefined): void;
+  public show(column?: any, preserveFocus?: any): void {}
+  public hide(): void {}
+  public dispose(): void {}
+}
```

Comment:
Can we reuse the mock defined in `channelService.test.ts` already? Maybe with a common mocks file.

Reply:
buena idea, combined the mocks and the stub of vscode